src/D_EcobeeHouse1.json

```diff
@@ -35,10 +35,10 @@
     ],
     "inScene": "1",
     "DisplayStatus": {
-        "Service": "urn:upnp-org:serviceId:SwitchPower1",
-        "Variable": "Status",
-        "MinValue": "0",
-        "MaxValue": "1"
+        "Service": "urn:ecobee-com:serviceId:Ecobee1",
+        "Variable": "currentClimateRef",
+        "MinValue": "home",
+        "MaxValue": "smart1"
```

Comment:
You are correct. I am assuming here that smart1 will always be "vacation" to match what is on the vera and it may not always be true. For some user this mode may well be smart2 or smart3. I will document this. Thanks